root/io/compression/MainEvent.cxx

```diff
@@ -307,7 +307,11 @@ int main(int argc, char **argv)
       else if (comp == 101) expectedSize = 1254808;
       else if (comp == 201) expectedSize = 1111230;
       else if (comp == 301) expectedSize = 1265018;
+#ifdef R__HAS_LZ4
```

Comment:
@pcanal I was inspired by code in ROOTConfigure.cmake `if(lz4) set(haslz4compression define)`. Do you prefer to change it?